src/functions/enclose.js

```diff
@@ -46,15 +46,24 @@ const htmlBuilder = (group, options) => {
 
         // Add vertical padding
         let vertPad = 0;
+        let ruleThickness = 0;
         // ref: LaTeX source2e: \fboxsep = 3pt;  \fboxrule = .4pt
```

Comment:
Could you give these values variable names?

Reply:
Done.